presto-main/src/main/resources/webapp/src/package.json

```diff
@@ -19,12 +19,11 @@
   },
   "dependencies": {
     "d3": "^5.7.0",
-    "dagre-d3": "^0.6.1",
+    "dagre-d3": "^0.6.4",
     "react": "18.2.0",
     "react-data-table-component": "^7.5.4",
     "react-dom": "18.2.0",
-    "styled-components": "^6.0.8",
-    "vis-timeline": "^7.7.2"
```

Comment:
`vis-timeline` can't be removed. It's used by most of the pages which contain time charts.